components/patient-tools/src/main/java/org/phenotips/tools/PhenotypeMappingService.java

```diff
@@ -226,13 +228,9 @@ private Object getMapping(String mappingName)
         DocumentReference mappingDoc = getMappingDocument();
         Object result = getMapping(mappingDoc, mappingName);
         if (result == null) {
-            Map<String, Object> mappings;
             try {
-                if (this.bridge.getDocumentContentForDefaultLanguage(mappingDoc).contains("{{velocity")) {
-                    mappings = parseVelocityMapping(mappingDoc);
-                } else {
-                    mappings = parseGroovyMapping(mappingDoc);
-                }
```

Comment:
Don't remove this code, backwards compatibility is important. Existing users might have custom mappings still using the old syntax, which will get broken after an upgrade. Add a new branch on top, checking if the content starts with `{`.